README.md

```diff
@@ -1,6 +1,10 @@
 # vrsus
+### [Inception deck](https://docs.library.ucla.edu/display/DEV/Build+Your+Own+Blacklight+%28BYOB%29+Inception+Deck) | [Product page](https://docs.library.ucla.edu/display/DEV/Ursus+v2)
 
 ## Project setup
+
+<details><summary>SETUP</summary>
```

Comment:
This markup is interesting to know about, but I question the usefulness of hiding the NPM commands (to me, the most important info in the README at this point) by default, requiring a user to click in order to show them. I also feel like because the README isn't very long right now, we don't need to worry about saving space yet.